docs/conf.py

```diff
@@ -71,7 +72,7 @@
 
 # General information about the project.
 project = 'LightGBM'
-copyright = '2017, Microsoft Corporation'
+copyright = '%s, Microsoft Corporation' %(str(datetime.datetime.now().year))
```

Comment:
We don't check PEP8 in docs , but I think it's better to avoid making errors in the new code.
Here is `E225 missing whitespace around operator` and outer brackets are excess.